src/kirby/components/list/list.component.ts

```diff
@@ -49,24 +43,27 @@ export class ListComponent implements OnInit {
 
   isSectionsEnabled: boolean;
   isSelectable: boolean;
+  isLoading: boolean;
+  isLoadOnDemandEnabled: boolean;
 
-  constructor() {}
+  constructor(private listHelper: ListHelper) {}
 
-  ngOnInit() {
+  ngOnInit(): void {
     if (this.getSectionName) {
       this.isSectionsEnabled = true;
     }
     if (this.listItemTemplate) {
       console.warn('kirbyListItem is deprecated and will be removed in future versions of Kirby');
     }
     this.isSelectable = this.itemSelect.observers.length > 0;
+    this.isLoadOnDemandEnabled = this.loadOnDemand.observers.length > 0;
   }
 
-  onItemClick(row: any): void {
-    this.itemSelect.emit(row);
+  onItemSelect(selectedItem: any): void {
+    this.itemSelect.emit(selectedItem);
   }
 
-  onItemTap(selectedItem: any): void {
-    this.itemSelect.emit(selectedItem);
+  onLoadOnDemand(args?: LoadOnDemandEventData): void {
```

Comment:
```suggestion
  onLoadOnDemand(event?: LoadOnDemandEvent): void {
```

Reply:
There is a other interface in this file called `LoadOnDemandEvent`. 

This `LoadOnDemandEventData` is ment as a wrapper for {N}'s `LoadOnDemandListViewEventData` interface.